src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  const threeDaysDiscount = 20;
+  const sevenDaysDiscount = 50;
+  const totalCost = days * costPerDay;
+
+  const discount = days >= 3
+    ? days < 7
```

Comment:
and also move 3 and 7 to constants too. because now we should gues what do they mean